packages/battery/battery/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: battery
 description: Flutter plugin for accessing information about the battery state
   (full, charging, discharging) on Android and iOS.
 homepage: https://github.com/flutter/plugins/tree/master/packages/battery/battery
-version: 1.0.10
+version: 2.0.0-nullsafety
```

Comment:
`2.0.0-nullsafety.0` maybe?

Reply:
It looks like the standard for this repo is to not add the `.0` at the end: 

https://github.com/flutter/plugins/blob/master/packages/webview_flutter/CHANGELOG.md#200-nullsafety
https://github.com/flutter/plugins/blob/master/packages/plugin_platform_interface/CHANGELOG.md#110-nullsafety

It would probably be best to follow the same pattern unless @amirh or @cyanglaz suggest something different.

Reply:
If this is the adopted style then ok.

However, it is a common mistake to interchange between `suffix` and `suffix.0` while the former intended to read `any version with the suffix` and the later `this is version 0 for this suffix`. Omitting `suffix` in the version effectively creates an implicit `suffix.0` version and looks ambiguous.